docs/yellowpaper/sections/05-HOPR-incentivization-mechanism/01-probabilistic-payments.tex

```diff
@@ -1,66 +1,108 @@
+claim tickets independently from each other.
 
 \begin{align}
- value ( ACK_i )  &  =value ( ACK_j ) \quad for \quad i,j\in \{1,n\}
+value ( ACK_i )  &  =value ( ACK_j ) \quad for \quad i,j\in \{1,n\}
 \end{align}
-If we assume the price stays the same. There is no added value in pretending packet loss or intentionally changing the order in which packets are processed.
 
-\subsubsection{Channel Management}
-Initially, each payment channel in HOPR is CLOSED which means that in order to transfer packets, those channels have to be opened. There are four distinct channel statuses which are represented in the following scheme:
+If we assume the price stays the same, there is no added value in pretending
+packet loss or intentionally changing the order in which packets are processed.
+On the contrary a node would reduce its potential payouts when not forwarding
+packets as fast as possible or not at all.
+
+\subsection{Payment Channel Management}
+\label{sec:paymentchannelmanagement}
+
+Initially, each payment channel is \textit{Closed}. For node $A$ to be able to
+transfer packets to node $B$ it has to open a payment channel first. There are
+four distinct payment channel states which are represented in the following
+scheme:
 
 \begin{figure}[H]
     \centering
     \includegraphics[width=9cm,height=9cm,keepaspectratio]{../yellowpaper/images/states2.png}
-\label{fig:channel statuses}
-    \caption{Channel statuses}
+    \label{fig:payment channel states}
+    \caption{Payment channel states}
 \end{figure}
 
-\paragraph{Opening a channel} Nodes can OPEN channels to other nodes by the following:
-A calls the method \textit{fundChannelMulti()} such that:
-$$fundChannelMulti(A: <\lambda>, B:<\mu> )$$ where $\lambda$ and $\mu$ are the amounts to be staked by the sender A for A and B. This function opens two unidirectional channels $A\rightarrow B$ and $B\rightarrow A$ if both staked amounts are specified and are $>0$. If only $\lambda>0$,  then only one channel is opened $A\rightarrow B$ where $A$ is the sender and $B$ is the recipient. $B$ in this case can call the same function again specifying that $\mu>0$ and $\lambda=0$ which will open the channel $B\rightarrow A$.
-Both channels are then on state \textit{WAITING\_FOR\_COMMITMENT}.
-\\~\\The destination address of the channel must now set an on-chain commitment in order for the channel between both parties to be open.
-The channel destination (let's say it's B) can now call \textit{bumpChannel()} function to make a new set of commitments towards this channel. This call will trigger an onchain event \textit{ChannelIsOpened} and bumbs the ticket epoch to ensure tickets with the previous epoch are invalidated
+\paragraph{Opening a channel} can be done by node $A$ by transferring funds to
+the payment channels contract \textit{HoprChannels} and include the following
+\textit{userdata}:
+
+$$[A: address, B: address, \lambda: uint8, \mu: uint8], \mu = 0$$
```

Comment:
the staking value given to B is not equal to 0 since A will stake for both A and B and that would open 2 unidirectional channels from A to B and from B to A. Check here https://github.com/hoprnet/hoprnet/blob/a691b7a1d0f636b7667e0f5c24be3b62aba0903c/packages/ethereum/contracts/HoprChannels.sol#L366

Reply:
Correct. While this is possible within the contract it is not used by the protocol implementation. Node A will only create a A->B channel, and not fund the B->A channel. 

See https://github.com/hoprnet/hoprnet/blob/master/packages/core-ethereum/src/ethereum.ts#L223